Formula/influxdb.rb

```diff
@@ -37,8 +37,8 @@ class Influxdb < Formula
   # NOTE: The version/URL here is specified in scripts/fetch-ui-assets.sh in influxdb.
   # If you're upgrading to a newer influxdb version, check to see if this needs upgraded too.
   resource "ui-assets" do
-    url "https://github.com/influxdata/ui/releases/download/OSS-v2.0.9/build.tar.gz"
-    sha256 "ace380b5bd6abef9aa0ca16e95900052b9520399a7b3311a0c366a5d98ad400d"
+    url "https://github.com/influxdata/ui/releases/download/OSS-2.1.2/build.tar.gz"
```

Comment:
Note that the UI version numbers do not necessarily match up 1:1 with the influxdb version, and the tags aren't specifically semver. See https://github.com/influxdata/influxdb/blob/657e1839de9e8a734abad1207ca28e7d02444207/scripts/fetch-ui-assets.sh#L21-L26